src/main/java/greedy/greedybot/presentation/jda/listener/ReviewMatchListener.java

```diff
@@ -0,0 +1,148 @@
+package greedy.greedybot.presentation.jda.listener;
+
+import greedy.greedybot.application.matching.MatchingService;
+import greedy.greedybot.application.matching.dto.MatchingResultAnnouncement;
+import net.dv8tion.jda.api.events.interaction.command.CommandAutoCompleteInteractionEvent;
+import net.dv8tion.jda.api.events.interaction.command.SlashCommandInteractionEvent;
+import net.dv8tion.jda.api.interactions.commands.Command;
+import net.dv8tion.jda.api.interactions.commands.OptionMapping;
+import net.dv8tion.jda.api.interactions.commands.OptionType;
+import net.dv8tion.jda.api.interactions.commands.build.Commands;
+import net.dv8tion.jda.api.interactions.commands.build.SlashCommandData;
+import org.jetbrains.annotations.NotNull;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+@Component
+public class ReviewMatchListener implements AutoCompleteInteractionListener {
+
+    private static final Logger log = LoggerFactory.getLogger(ReviewMatchListener.class);
+    private static final String[] reviewees = new String[]{
+            "BE-1기: 남해윤, 안금서, 신지훈, 정상희, 신혜빈, 김의진, 황승준",
+            "FE-1기: 송혜정, 김준수",
+    };
+    private static final String[] reviewers = new String[]{
+            "BE-1기: 원태연, 이승용, 송은우, 백경환, 김주환, 조승현",
+            "FE-1기: 김범수, 김의천",
+    };
+
+
+    private final MatchingService matchingService;
+
+
+    public ReviewMatchListener(
+            final MatchingService matchingService
+    ) {
+        this.matchingService = matchingService;
+    }
+
+    @Override
+    public String getCommandName() {
+        return "review";
```

Comment:
```suggestion
        return "review-match";
```
액션이 포함된 커맨드면 더 좋을 것 같아요!